src/components/primitive/Text/text.component.utils.ts

```diff
@@ -1,56 +1,48 @@
-import styled from 'styled-components';
 import { FONT_SIZE, ALIGNMENTS } from '../../../utils/typography';
-import { Variants, FontSizes, Align, StyledTextProps } from './text.props';
 
 // styled component doesn't support varients natively- so we are using switch here.
 
 const handleColor = (variant: string, theme: any) => {
   switch (variant) {
-    case Variants.PRIMARY:
+    case 'primary':
```

Comment:
Why are these things hard-coded? The spec was to remove such functions and organize them based on variants right?

Reply:
removed all old implementation and organized according to Variants.